sim/paladin/judgement.go

```diff
@@ -45,33 +47,99 @@ func (paladin *Paladin) registerJudgement() {
 			//}
 			// paladin.currentSeal.Deactivate(sim)
 
+			thisJudgement := 0 // Seals Judged this Judgement
```

Comment:
Could you create a bitflag enum for the judgements? Similar to warrior stances in `stances.go`

Reply:
done